templates/embedded-editor.html

```diff
@@ -15,9 +19,9 @@
         <link rel="stylesheet" href="{{static('/fontawesome/css/all.min.css')}}"/>
     </head>
     <body class="flex flex-col gap-2 h-screen" dir="{{ g.dir }}">
-        <div id="editor-area" class="relative grid grid-cols-1 {% if run %}md:grid-cols-2{% endif %} md:grid-flow-row-dense gap-2">
+        <div id="editor-area" class="relative grid grid-cols-1 {% if run and not fullWidth %}grid-cols-2{% endif %} md:grid-flow-row-dense gap-2">
```

Comment:
Did we change how we're specifying editor height again? I recently rejiggered all of that in this PR: https://github.com/hedyorg/hedy/pull/4915